index.html

```diff
@@ -140,10 +140,9 @@
               <img src="./img/img-slider.jpg" alt="Слайдер Товарів" />
             </div>
             <div class="slider__bottom">
-              <span class="slider__bottom__text"
-                >Каталог постільних <br />комплектів</span
-              >
-
+              <div class="slider__bottom__text">
+                <span>Каталог постільних комплектів</span>
```

Comment:
why `span` is needed here? 🤔 
I would put everything in `<div>`

Reply:
Agree, no sense use `span`